hydra-node/json-schemas/api.yaml

```diff
@@ -755,19 +738,60 @@ components:
           timestamp:
             $ref: "#/components/schemas/UTCTime"
 
+    DraftCommitTxRequest:
+      title: DraftCommitTxRequest
+      description: |
+        Provide utxos that will be used to draft a commit transaction that will be sent back to the user.
+        Script utxos must provide a witness, regulars can ommit this.
+      payload:
+        type: object
+        required:
+          - utxos
+        additionalProperties: false
+        properties:
+          utxos:
+            type: array
+            items:
+              $ref: "#/components/schemas/UTxO"
+        example:
+          {
+            "utxos": [
+              {
+                "0406060506030602040508060506060306050406020207000508040704040203#89": {
+                  "address": "addr_test1vz66ue36465w2qq40005h2hadad6pnjht8mu6sgplsfj74q9pm4f4",
+                  "value": {
+                    "lovelace": 7620669
+                  }
+                },
```

Comment:
This schema is incomplete. I suspect this example will not parse as `witness` is not really optional right now.

Also the `$ref: ...` above need to point not to `UTxO`, but to something that is `UTxO` + the `witness` field

Reply:
Indeed this example is not parseable.

Do know how to make `witness` really optional?

[Pushed](https://github.com/input-output-hk/hydra/pull/903/commits/b27bf8eb3aa9b10be09cf5be5d855805a34753be) new types with witnesses to not point to `UTxO`.

Reply:
Left a hint on the `FromJSON`, I would suggest you update the example here to remove `"witness":null`